services/bettersocial/post/createPostV3.js

```diff
@@ -330,7 +330,7 @@ const BetterSocialCreatePostV3 = async (req, isAnonimous = true) => {
     else post = await Getstream.feed.createPost(req?.token, data);
 
     // Upvote post before sending the success message
-    await upVote(post?.id, req.token, req?.userId);
+    await upVote(post?.id, req.token, userDetail?.user_id);
     countProcess(post?.id, {upvote_count: +1}, {upvote_count: 1});
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `async` in `forEach` might not work as expected because `forEach` doesn't wait for promises. If you want to perform asynchronous operations for each item in an array and wait for all of them to complete, consider using `Promise.all` with `map`.

```diff
-    body.tagUsers?.forEach(async (user_id) => {
+    if (body.tagUsers) {
+        await Promise.all(body.tagUsers.map(async (user_id) => {
+            // Your async operation here
+        }));
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->